Rock/PersonProfile/Badge/InDataView.cs

```diff
@@ -0,0 +1,75 @@
+﻿// <copyright>
+// Copyright by the Spark Development Network
+//
+// Licensed under the Rock Community License (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.rockrms.com/license
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+//
+using System.ComponentModel;
+using System.ComponentModel.Composition;
+using System.IO;
+using System.Linq;
+using System.Data.Entity;
+
+using Rock.Attribute;
+using Rock.Model;
+using Rock.Web.UI.Controls;
+using Rock.Data;
+using System.Collections.Generic;
+using System.Data;
+using System;
+using System.Diagnostics;
+using Rock.Web.Cache;
+
+namespace Rock.PersonProfile.Badge
+{
+
+    /// <summary>
+    /// 
+    /// </summary>
+    [Description( "Displays if the person has an alert note." )]
+    [Export( typeof( BadgeComponent ) )]
+    [ExportMetadata( "ComponentName", "In Data View" )]
+
+    [DataViewField( "Data View", "The dataview to look at.", order: 0 )]
```

Comment:
Missing arg entityTypeName: "Rock.Model.Person" which currently prevents the list of available DataViews from being populated. Also set the required field to be true:

```csharp
[DataViewField( "Data View", "The dataview to use as the source for the query. Only those people in the DataView will be given the badge.", true, entityTypeName: "Rock.Model.Person", order: 0 )]
```